android/app/src/main/AndroidManifest.xml

```diff
@@ -28,7 +27,8 @@
         android:name=".MainActivity"
         android:label="@string/app_name"
         android:configChanges="keyboard|keyboardHidden|orientation|screenSize"
-        android:windowSoftInputMode="adjustResize">
+        android:windowSoftInputMode="adjustResize"
+        android:launchMode="singleInstance">
```

Comment:
This is so weird, but ok cool. Not sure why this would not be the default. Related to this... I've noticed when i tap a deep link from Slack, the Textile activity acutally starts up "inside" the Slack process. It's very strange. Anyone else?

Reply:
I've fixed that for Android (that is what this fix does exactly).
For iOS I don't know the solution yet.

Reply:
I didn't think it' a problem in iOS. No?

Reply:
doh, sorry, added new comment on wrong place